### PR TITLE
Clarify support status

### PR DESCRIPTION
**Pull request by mit-mit**
@mraleph is this a good way of phrasing it?

### CODE REVIEW

File: src/server/c-interop-native-extensions.md [hunk 1]
```diff
@@ -6,10 +6,15 @@ description: The original way for command-line Dart apps to call C/C++ functions
 _Written by William Hesse <br>
 May 2012_
 
-The extension mechanism discussed in this page is for
-deep integration of the VM.
+{{site.alert.note}}
+The extension mechanism discussed in this page is for deep integration of the VM
+by custom Dart embedders. This extension mechanism is **deprecated**, and may be
```
Comment (mraleph) [inline src/server/c-interop-native-extensions.md:11]: I still really don't like the world `**deprecated**` I would say `**deprecated** for non embedding use cases`

**Review by mraleph — APPROVED** [review]

**Comment by mit-mit** [comment]
Do you like **unsupported** better?

File: src/server/c-interop-native-extensions.md (hunk 1 above)
Comment (mit-mit) [reply src/server/c-interop-native-extensions.md:11]: Do you like **unsupported** better?

Comment (mraleph) [reply src/server/c-interop-native-extensions.md:11]: I like `**unsupported** for extension purposes` best.

**Comment by kwalrath** [comment]
Staged:
https://kw-staging-dartlang-2.firebaseapp.com/server/c-interop-native-extensions

**Pull request merged**